invenio_access/upgrades/access_2015_05_06_accROLE_accACTION_accARGUMENT_id.py

```diff
@@ -70,7 +70,9 @@ def do_upgrade():
         CHANGE COLUMN "id_accARGUMENT" "id_accARGUMENT" INT(15) NULL ,
         CHANGE COLUMN "argumentlistid" "argumentlistid" MEDIUMINT(8) NULL ,
         ADD COLUMN "id" INT(15) UNSIGNED NOT NULL AUTO_INCREMENT,
-        DROP PRIMARY KEY,
+        DROP KEY id_accROLE,
```

Comment:
```
mysql> describe accROLE_accACTION_accARGUMENT;
+----------------+--------------+------+-----+---------+-------+
| Field          | Type         | Null | Key | Default | Extra |
+----------------+--------------+------+-----+---------+-------+
| id_accROLE     | int(15)      | YES  | MUL | NULL    |       |
| id_accACTION   | int(15)      | YES  | MUL | NULL    |       |
| id_accARGUMENT | int(15)      | YES  | MUL | NULL    |       |
| argumentlistid | mediumint(8) | YES  |     | NULL    |       |
+----------------+--------------+------+-----+---------+-------+
4 rows in set (0.00 sec)
```
And this is what happen if I run the upgrade as it is now (actually it fails silently)

```
mysql> ALTER TABLE "accROLE_accACTION_accARGUMENT"
    ->  CHANGE COLUMN "id_accROLE" "id_accROLE" INT(15) UNSIGNED NULL ,
    ->  CHANGE COLUMN "id_accACTION" "id_accACTION" INT(15) UNSIGNED NULL ,
    ->  CHANGE COLUMN "id_accARGUMENT" "id_accARGUMENT" INT(15) NULL ,
    ->  CHANGE COLUMN "argumentlistid" "argumentlistid" MEDIUMINT(8) NULL ,
    ->  ADD COLUMN "id" INT(15) UNSIGNED NOT NULL AUTO_INCREMENT,
    ->  DROP PRIMARY KEY,
    ->  ADD PRIMARY KEY ("id");
ERROR 1091 (42000): Can't DROP 'PRIMARY'; check that column/key exists
```